src/errors.ts

```diff
@@ -13,6 +10,8 @@ export const enum ErrorCode {
     BadRequest,
     RateLimited,
     AuthServerError,
+    UserNotFound = 'UNOTFOUND',
+    AccessDenied = 'UACCESS',
```

Comment:
Mixing numbers and strings in this enum is weird. It's probably also not going to be always correct, e.g. I don't think those can be used in place of codes in [IErrorPacket](https://github.com/mixer/client-node/blob/f3ea80820ba199742760bfb68a5ae9b4570f540d/src/ws/Reply.ts#L8)